src/client/kotlin/tech/thatgravyboat/skyblockapi/api/profile/equipment/EquipmentAPI.kt

```diff
@@ -0,0 +1,74 @@
+package tech.thatgravyboat.skyblockapi.api.profile.equipment
+
+import net.minecraft.world.item.ItemStack
+import net.minecraft.world.item.Items
+import tech.thatgravyboat.skyblockapi.api.data.stored.EquipmentStorage
+import tech.thatgravyboat.skyblockapi.api.datatype.DataTypes
+import tech.thatgravyboat.skyblockapi.api.datatype.getData
+import tech.thatgravyboat.skyblockapi.api.events.base.Subscription
+import tech.thatgravyboat.skyblockapi.api.events.chat.ChatReceivedEvent
+import tech.thatgravyboat.skyblockapi.api.events.level.RightClickEvent
+import tech.thatgravyboat.skyblockapi.api.events.screen.InventoryChangeEvent
+import tech.thatgravyboat.skyblockapi.api.events.screen.InventoryFullyLoadedEvent
+import tech.thatgravyboat.skyblockapi.modules.Module
+import tech.thatgravyboat.skyblockapi.utils.regex.RegexGroup
+import tech.thatgravyboat.skyblockapi.utils.regex.RegexUtils.find
+import tech.thatgravyboat.skyblockapi.utils.text.TextProperties.stripped
+
+@Module
+object EquipmentAPI {
+
+    private val inventoryNameRegex = RegexGroup.INVENTORY.group("equipment").create(
+        "title",
+        "Your Equipment and Stats"
+    )
+
+    private val chatEquipRegex = RegexGroup.CHAT.group("equipment").create(
+        "equip",
+        "You equipped a (?<item>.+)"
```

Comment:
Needs an ! at the end
```suggestion
        "You equipped a (?<item>.+)!"
```